lisa/tools/wget.py

```diff
@@ -44,6 +44,11 @@ def get(
         matched_result = self.__pattern_path.match(command_result.stdout)
         if matched_result:
             download_file_path = matched_result.group("path")
+        else:
+            raise LisaException(
+                f"cannot find file path in output of '{run_command}', "
+                f"it may cause by download failed or pattern mismatch."
```

Comment:
If the output has only one line, it can be included in the exception message. So, we don't need look into log file when it happens. But if it's many lines, we cannot put too much information to the message.

Reply:
what action should I take?

Reply:
Try to output the raw output to help understand what happens.